common/changes/@itwin/breakdown-trees-react/jake-core-4.0-update_2023-05-22-17-36.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/breakdown-trees-react",
+      "comment": "Updated to iTwin.js Core 4.0 release",
```

Comment:
Don't need this comment since we'll get the changelog entry from the pre release we did with the core 4.0 version

Reply:
removed comment